### PR TITLE
Pull in reads for 29 1000 Genomes samples

### LINKED ISSUE #9: Pull in a small sample of 1000G low coverage WGS data
I was going to pull down 50 low coverage WGS files, for testing with [avocado](https://www.github.com/bigdatagenomics/avocado). I was going to pull down:

```
HG00126
HG00173
HG00258
HG00369
HG00583
HG00641
HG00664
HG01070
HG01079
HG01346
HG01495
HG01629
NA10836
NA11829
NA12234
NA12375
NA17978
NA18489
NA18546
NA18549
NA18633
NA18702
NA18773
NA18789
NA18989
NA19070
NA19107
NA19138
NA19153
NA19173
NA19175
NA19180
NA19198
NA19240
NA19564
NA19568
NA19729
NA19913
NA20288
NA20364
NA20761
NA20772
NA20864
NA20886
NA21297
NA21344
NA21369
NA21371
NA21494
NA21636
```

These samples give us a population distribution of:

```
   3 ASW
   4 CEU
   5 CHB
   2 CHD
   2 CHS
   2 CLM
   2 FIN
   2 GBR
   2 GIH
   1 IBS
   4 JPT
   6 MKK
   1 MXL
   3 PUR
   2 TSI
   9 YRI
```

### PR DESCRIPTION
**Pull request by fnothaft**
Resolves #9. @laserson , do you want this opened against master or experiment?

### CODE REVIEW

**Comment by laserson** [comment]
master.

experiment is just to push WIP changes.  The reason is that when provisioning the cluster, the fabric code checks out eggo on the worker machines, so there is a switch that will checkout the experiment branch.  is there a principled way to do this?  otherwise, testing features/debugging can be annoying.

**Comment by laserson** [comment]
oh, the state in my head is weird...what if you rebase against master?

File: eggo/fabric_util.py [hunk 1]
```diff
@@ -106,6 +105,8 @@ def _install_eggo():
     with cd('eggo'):
         if os.environ.get('EGGO_EXP', 'FALSE') == 'TRUE':
             run('git checkout -b experiment origin/experiment')
+        elif os.environ.get('EGGO_FRANK_EXP', 'FALSE') == 'TRUE':
+            run('git checkout -b 1000g-29-samples origin/1000g-29-samples')
```
Comment (laserson) [inline eggo/fabric_util.py]: Yes, what's the proper way to deal with this.  This is janky

File: registry/1000g-subset.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+    "name": "1000g-subset",
+    "description": "Subset of 1000G Low Coverage WGS Data",
+    "target": "1000g-subset/reads",
```
Comment (laserson) [inline registry/1000g-subset.json:4]: maybe, `1kg-subset/reads` or `1kg/<insert_description>`

Actually, in the spec doc, we should talk about optimal "directory" structure for the S3 data.

Comment (fnothaft) [inline registry/1000g-subset.json:4]: Deffo; will make the change. I've already pushed the data into S3; I'll move it there as well.

File: eggo/fabric_util.py (hunk 1 above)
Comment (fnothaft) [inline eggo/fabric_util.py]: Yeah, I wasn't loving it. What _could_ work is allowing users to pass flags on the command line to override the fork and branch.

**Comment by fnothaft** [comment]
This is rebased on master now, but still has the two WIP commits from the other branch. Do you want me to drop those two commits?

**Comment by laserson** [comment]
Ok, rebase this on master again.  That should clear things up.  Sorry...

File: eggo/fabric_util.py (hunk 1 above)
Comment (laserson) [inline eggo/fabric_util.py]: command line config better than env vars `EGGO_FORK` and `EGGO_BRANCH`?

Comment (fnothaft) [inline eggo/fabric_util.py]: Ah, I like `EGGO_FORK` `EGGO_BRANCH` better.

**Comment by fnothaft** [comment]
Rebased.

File: eggo/config.py [hunk 3]
```diff
@@ -18,7 +18,6 @@
 
 from eggo.util import random_id
 
-
```
Comment (laserson) [inline eggo/config.py:21]: nit: I think pep8 will yell about this, as it likes 2 lines between top-level items

File: eggo/dag.py [hunk 4]
```diff
@@ -231,7 +229,7 @@ def run(self):
         if format.lower() != 'vcf':
             raise ValueError("Expected 'vcf' format; got {0}".format(format))
 
-        # 1. Copy the data from S3 to the local ephemeral HDFS
```
Comment (laserson) [inline eggo/dag.py:234]: Undo this change

**Comment by laserson** [comment]
Looks great!  Just small nits.  Could you also squash to one commit?

**Comment by fnothaft** [comment]
Fixed the nits!

**Comment by fnothaft** [comment]
Closed by #15.
